src/test/java/tests/TC14_UserCanBookManyTicketAtATime.java

```diff
@@ -0,0 +1,46 @@
+package tests;
+
+import helper.DataHelper;
+import models.Account;
+import models.Ticket;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.BookTicketPage;
+import page_objects.HomePage;
+import page_objects.LoginPage;
+import page_objects.RegisterPage;
+
+public class TC14_UserCanBookManyTicketAtATime extends TestBase {
+    HomePage homePage = new HomePage();
+    RegisterPage registerPage = new RegisterPage();
+    LoginPage loginPage = new LoginPage();
+    BookTicketPage bookTicketPage = new BookTicketPage();
+
+    String departDate = DataHelper.getRandomDepartDate();
```

Comment:
we dont need to define `getRandomDepartDate` in DataHelper.

Reply:
I have fixed it